example/lib/hms_app_settings.dart

```diff
@@ -490,8 +494,8 @@ class _HMSAppSettingsState extends State<HMSAppSettings> {
                     ),
                     trailing: Text(
                       Platform.isAndroid
-                          ? HMSSDKConstants.androidSDKVersion
-                          : HMSSDKConstants.iOSSDKVersion,
+                          ? versions["android"]??""
```

Comment:
Similar to the above. If the version could not be found, there should be an Error log. All SDK versions should always have the correct versions defined.